packages/api-server/README.md

```diff
@@ -29,6 +29,10 @@ e.g.
 ```bash
 RMF_API_SERVER_CONFIG='my_config.py' rmf_api_server
 ```
+To run the api-server with PostgreSQL, assuming it has been set up to listen on 127.0.0.1:5432 with user `postgres` and password `postgres`:
+```
+RMF_API_SERVER_CONFIG=api_server/psql_config.py rmf_api_server
```

Comment:
I think this needs to be updated with the new path.

Reply:
1ea7f6e7264238858eb53280a3cb46e0bae64c92 thanks for catching this!